client/public/index.html

```diff
@@ -2,7 +2,7 @@
 <html lang="en">
   <head>
     <meta charset="utf-8" />
-    <link rel="shortcut icon" href="%PUBLIC_URL%/favicon.ico" />
+    <link rel="shortcut icon" href="%PUBLIC_URL%/chess-favicon.ico" />
```

Comment:
Make sure there are no licensing issues while using images.
Mentioning, just in case.

Reply:
Yes, downloaded the free .ico file